sim/common/bitmanip.h

```diff
@@ -3,6 +3,7 @@
 #include <cstdint>
 #include <algorithm>
 #include <assert.h>
+#include "xlen.h"
```

Comment:
this file is a general library, should not have xlen dependency.
remove include "xlen.h"